kitsune/questions/static/questions/js/components/AAQApp.jsx

```diff
@@ -39,17 +40,30 @@ export default class AAQApp extends React.Component {
       validationErrors: QuestionEditStore.getValidationErrors(),
       questionState: QuestionEditStore.getState(),
       userAuth: UserAuthStore.getAll(),
-      troubleshooting: TroubleshootingDataStore.getAll(),
+      troubleshooting: TroubleshootingDataStore.getAll()
     };
   }
 
+  setStep(step) {
+    this.setState({step: step});
+  }
+
   render() {
     return (
       <div className="AAQApp">
-        <ProductSelector {...this.state}/>
-        <TopicSelector {...this.state}/>
-        <TitleContentEditor {...this.state}/>
-        <UserAuth userAuth={this.state.userAuth}/>
+        {(() => {
+          switch (this.state.step) {
+            case 'topic':
+              return <TopicSelector {...this.state} setStep={this.setStep.bind(this)} next="title"/>;
```

Comment:
You could do `<TopicSelector {...this.state} nextStep={this.setStep.bind(this, "title")}/>`. Binding the argument into the parameter makes the child component simpler, since it just has a single function to call to move to the next step.